src/components/Tags/MultiFilterStyleTagSelector.tsx

```
@@ -150,54 +150,85 @@ export function MultiFilterStyleTagSelector({
+          className="flex items-center gap-3 p-3 rounded-lg hover:bg-gray-50 cursor-pointer"
+          onClick={() => {
+            if (tag.has_children) {
+              handleMobileGroupClick(tag);
+            } else {
+              handleSelect(tag);
+            }
+          }}
+        >
+          <button
+            type="button"
+            role="checkbox"
```

Biome (2.3.14)

[error] The elements with this role can be changed to the following elements:
<input type="checkbox">

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)